pkg/cmd/openshift-controller-manager/cmd.go

```diff
@@ -146,17 +145,6 @@ func (o *OpenShiftControllerManager) RunControllerManager() error {
 	if err != nil {
 		return err
 	}
-
-	validationResults := validation.ValidateMasterConfig(masterConfig, nil)
```

Comment:
why is this not needed?

Reply:
> why is this not needed?

never valid except from integration test.